js/viz/translators/translator2d.js

```diff
@@ -149,9 +150,19 @@ function isEqualRange(range) {
     const businessRange = that.getBusinessRange();
     const canvasOptions = getCanvasBounds(businessRange);
     const correctionPrecision = getEqualityCorrection(businessRange) / 100;
+    const comparingRange = range && new rangeModule.Range(range);
```

Comment:
range && - extra operation. It's exist in constructor of Range

Reply:
and in 'if(comparingRange' use 'range' instead for this checking

Reply:
new rangeModule.Range(range) returns object anyway

Reply:
done